src/main/java/frc/robot/Subsystems/Climber.java

```diff
@@ -1,4 +1,54 @@
 package frc.robot.Subsystems;
 
-public class Climber {
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+
+public class Climber extends MotoredGenericSubsystem {
+    private static final double MIN_SPEED = -0.6;
+    private static final double MAX_SPEED = 0.6;
+
+    private static Climber instance;
+    private static boolean counted = false;
+    private static int magnets = 0;
+    private static DigitalInput digitalInput;
+
+
+    public static Climber getInstance() {
+        if (instance == null) {
+            instance = new Climber(new WPI_VictorSPX(-1), new WPI_VictorSPX(-1), new DigitalInput(-1));
```

Comment:
before calling the constructor you should do the `victor.follow `and the `victor.setInverted`